test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1

```diff
@@ -0,0 +1,316 @@
+        CompressedFiles = "Microsoft.PowerShell.Host_56d66100-99a0-4ffc-a12d-eee9a6718aef_en-US_helpcontent$extension"
+    }
+
+    "Microsoft.PowerShell.LocalAccounts" = @{
+        HelpFiles = "Microsoft.Powershell.LocalAccounts.dll-help.xml"
+        HelpInfoFiles = "Microsoft.PowerShell.LocalAccounts_8e362604-2c0b-448f-a414-a6a690a644e2_HelpInfo.xml"
+        CompressedFiles = "Microsoft.PowerShell.LocalAccounts_8e362604-2c0b-448f-a414-a6a690a644e2_en-US_HelpContent$extension"
+    }
+
+    "Microsoft.PowerShell.Management" = @{
+        HelpFiles = "Microsoft.PowerShell.Commands.Management.dll-help.xml"
+        HelpInfoFiles = "Microsoft.PowerShell.Management_eefcb906-b326-4e99-9f54-8b4bb6ef3c6d_HelpInfo.xml"
+        CompressedFiles = "Microsoft.PowerShell.Management_eefcb906-b326-4e99-9f54-8b4bb6ef3c6d_en-US_helpcontent$extension"
+    }
+
+    "Microsoft.PowerShell.Security" = @{
+        HelpFiles = "Microsoft.PowerShell.Security.dll-help.xml"
+        HelpInfoFiles = "Microsoft.PowerShell.Security_a94c8c7e-9810-47c0-b8af-65089c13a35a_HelpInfo.xml"
+        CompressedFiles = "Microsoft.PowerShell.Security_a94c8c7e-9810-47c0-b8af-65089c13a35a_en-US_helpcontent$extension"
+    }
+
+    "Microsoft.PowerShell.Utility" = @{
+        HelpFiles = "Microsoft.PowerShell.Commands.Utility.dll-Help.xml", "Microsoft.PowerShell.Utility-help.xml"
+        HelpInfoFiles = "Microsoft.PowerShell.Utility_1da87e53-152b-403e-98dc-74d7b4d63d59_HelpInfo.xml"
+        CompressedFiles = "Microsoft.PowerShell.Utility_1da87e53-152b-403e-98dc-74d7b4d63d59_en-US_helpcontent$extension"
+    }
+
+    "Microsoft.WSMan.Management" = @{
+        HelpFiles = "Microsoft.WSMan.Management.dll-help.xml"
+        HelpInfoFiles = "Microsoft.WsMan.Management_766204A6-330E-4263-A7AB-46C87AFC366C_HelpInfo.xml"
+        CompressedFiles = "Microsoft.WsMan.Management_766204A6-330E-4263-A7AB-46C87AFC366C_en-US_helpcontent$extension"
+    }
+
+    "PackageManagement" = @{
+        HelpFiles = "Microsoft.PowerShell.PackageManagement.dll-help.xml"
+        HelpInfoFiles = "PackageManagement_4ae9fd46-338a-459c-8186-07f910774cb8_HelpInfo.xml"
+        CompressedFiles = "PackageManagement_4ae9fd46-338a-459c-8186-07f910774cb8_en-US_helpcontent$extension"
+    }
+
+    "PowershellGet" = @{
+        HelpFiles = "PSGet.psm1-help.xml"
+        HelpInfoFiles = "PowershellGet_1d73a601-4a6c-43c5-ba3f-619b18bbb404_HelpInfo.xml"
+        CompressedFiles = "PowershellGet_1d73a601-4a6c-43c5-ba3f-619b18bbb404_en-US_helpcontent$extension"
+    }
+
+    "PSReadline" = @{
+        HelpFiles = "Microsoft.PowerShell.PSReadline.dll-help.xml"
+        HelpInfoFiles = "PSReadline_5714753b-2afd-4492-a5fd-01d9e2cff8b5_HelpInfo.xml"
+        CompressedFiles = "PSReadline_5714753b-2afd-4492-a5fd-01d9e2cff8b5_en-US_helpcontent$extension"
+    }
+}
+
+# These are the paths where the help content will be installed to.
+$getHelpContentPaths = @(
```

Comment:
This is separate logic -- $getHelpContentPaths are the paths for which we ensure that the help content is deleted before each test is run.

$expectedFiles contains the file names for the modules being tested. If we add $getHelpContentPaths to $expectedFiles, there will be a lot of repetition since the help content for most of the built-in modules is installed to $pshom\en-US.